pkg/kritis/metadata/metadata.go

```diff
@@ -27,14 +27,14 @@ type Fetcher interface {
 	Vulnerabilities(containerImage string) ([]Vulnerability, error)
 	// Create Attesatation Occurrence for an image.
 	CreateAttestationOccurence(note *grafeasv1beta1.Note,
```

Comment:
Instead of an attestation authority, this should accept a project name where to create occurrences.  There's no reason to create occurrences in the same project where the note is.

So the contract of the function should be something like:
Creates an attestation occurrence in project 'proj', attached to 'note'.  Created occurrence contains a signature by 'pgpSigningKey' over 'containerImage'.

Reply:
Looks like the latest reflects this recommendation.